example/sample/analytics/dataset.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
The `sample/docker-compose.yml` file needs `cbas` added to the list of services, otherwise, the presence of these files cause it to have errors.